lib/AST/ConformanceLookupTable.cpp

```diff
@@ -286,6 +286,10 @@ void ConformanceLookupTable::updateLookupTable(NominalTypeDecl *nominal,
           ASTContext &ctx = nominal->getASTContext();
           (void)evaluateOrDefault(
               ctx.evaluator, ExpandConformanceMacros{nominal}, { });
+
+          // Expand extension macros.
+          (void)evaluateOrDefault(
```

Comment:
This expansion is very "eager", in that we're expanding all extension macros even if they don't provide any conformances. I think it'll be okay for now.